metadata/pkg/service/service.go

```diff
@@ -23,7 +23,7 @@ import (
 
```

Comment:
why do you need this file, if there is no update??

Reply:
the metabucket model (from proto package) used in this file is different from what we are currently using for metadata collection. This causes errors while compiling code for metadata service. Hence, that part is commented for now.

Reply:
this issue is resolved and this file is unchanged.